airbyte-webapp/src/pages/SettingsPage/pages/ConnectorsPage/components/CreateConnectorModal.tsx

```diff
@@ -84,13 +80,21 @@ const ErrorText = styled.div`
   max-width: 400px;
 `;
 const validationSchema = yup.object().shape({
-  name: yup.string().required("form.empty.error"),
-  documentationUrl: yup.string().required("form.empty.error"),
-  dockerImageTag: yup.string().required("form.empty.error"),
-  dockerRepository: yup.string().required("form.empty.error"),
+  name: yup.string().trim().required("form.empty.error"),
+  documentationUrl: yup.string().trim().url("form.url.error").notRequired(),
```

Comment:
Note: this makes this field optional for OSS as well, which also does not appear to do anything with it beyond the link on the connectors list in settings + the docs panel.